lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
```

Comment:
Uhm... This line is there because it could work as before. Actually that was my first thought as well, but the old code what you accepted did the same thing. So my thought was after to check it that why not. Basically why should we so strict? This gem purpose is to send to the content to the Mandrill server via api. We don't have to filter anything just send through and then the api rejected if there is a problem. So I think this is ok, because if somebody forget to fill the content type then it will send it as html like before. Actually a normal text in the html doesn't cause any problem. But in the opposite case yes.